src/main/java/mate/academy/internet/shop/dao/jdbc/ItemDaoJdbcImpl.java

```diff
@@ -0,0 +1,102 @@
+package mate.academy.internet.shop.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import mate.academy.internet.shop.dao.ItemDao;
+import mate.academy.internet.shop.lib.Dao;
+import mate.academy.internet.shop.model.Item;
+import org.apache.log4j.Logger;
+
+@Dao
+public class ItemDaoJdbcImpl extends AbstractDao<Item> implements ItemDao {
+    private static Logger logger = Logger.getLogger(ItemDaoJdbcImpl.class);
+    private static String DB_Table = "storage.items";
```

Comment:
make it constant, please

Reply:
done. Please, explain why it is important here

Reply:
@IegorSoboliev There is no good reason to declare a field "public" and "static" without also declaring it "final". Most of the time this is a kludge to share a state among several objects. But with this approach, any object can do whatever it wants with the shared state, such as setting it to null.